docs/sources/setup-grafana/installation/debian/index.md

```diff
@@ -70,35 +72,7 @@ sudo apt-get install grafana
 sudo apt-get install grafana-enterprise
 ```
 
-### Install .deb package
```

Comment:
@DanCech , removed all this content because I think we can rely on the download page for guidance. Does that work?